lib/slaph_interface.cpp

```diff
@@ -0,0 +1,106 @@
+#include <quda.h>
```

Comment:
This should all go in `interface_quda.cpp`. I'm not opposed to refactoring the interface file at some point, it's very messy, but that's a separate discussion and we shouldn't change conventions here.

Reply:
[ec81a2e](https://github.com/lattice/quda/pull/1441/commits/ec81a2e9251f248f1abbb25d49e671f3266355ee)